### PR TITLE
feature/dependabot

### LINKED ISSUE #3: Introduce a dependency autoupdate tool

### PR DESCRIPTION
**Pull request by dirk39**
Close: #3

### CODE REVIEW

**Review by ludusrusso — APPROVED** [review]
LGTM

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    schedule:
+      interval: "weekly"
+    target-branch: "main"
+    open-pull-requests-limit: 1
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    target-branch: "main"
+    open-pull-requests-limit: 1
```
Comment (omissis) [inline .github/dependabot.yaml]: missing nl at eof

File: docs/arch/0007-dependency-alert-tool.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# 4. Dependency alert tool
```
Comment (omissis) [inline docs/arch/0007-dependency-alert-tool.md]:
```suggestion
# 7. Dependency alert tool
```

**Review by omissis — CHANGES_REQUESTED** [review]

File: docs/arch/0007-dependency-alert-tool.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# 7. Dependency alert tool
```
Comment (omissis) [inline docs/arch/0007-dependency-alert-tool.md]:
```suggestion
# 6. Dependency alert tool
```

File: docs/arch/0007-dependency-alert-tool.md
Comment (omissis) [inline docs/arch/0007-dependency-alert-tool.md:1]: I just noticed there's no ADR 6 :D We should rename this file to `docs/arch/0006-dependency-alert-tool.md`

**Review by omissis — CHANGES_REQUESTED** [review]

File: docs/arch/0007-dependency-alert-tool.md
Comment (dirk39) [reply docs/arch/0007-dependency-alert-tool.md:1]: I thought that the #13 comes with a new ADR 😓 Let me update

**Review by omissis — APPROVED** [review]

**Pull request merged**